tests/testthat/test-aaa_survival_prob.R

```diff
@@ -0,0 +1,256 @@
+test_that("survfit_summary_typestable() works for survival prob - unstratified (coxph)", {
```

Comment:
`survfit_summary_typestable()` is tested for a whole lot of different situations so that we don't have to test for those at a high level: stratified/unstratified, single/multiple observations, and single/multiple time points. 
glmnet returns survfit objects without several of the standard components (confidence intervals and standard errors) which is why those are being tested separately.